### PR TITLE
feat(core): slider horizon theme

### PR DESCRIPTION
**Pull request by N1XUS**
## Related Issue(s)
relates #7931

## Description
Added Horizon theme for the Slider component.

## Screenshots
image
image
image
image

##### During Implementation

1. Visual Testing:

-   [x] visual misalignments/updates
-   [x] check Light/Dark/HCB/HCW themes
-   [x] RTL/LTR - proper rendering and labeling
-   [x] responsiveness(resize)
-   [x] Content Density (Cozy/Compact/(Condensed))
-   [x] States - hover/disabled/focused/active/on click/selected/selected hover/press state
-   [x] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
-   [x] Mouse vs. Keyboard support
-   [x] Text Truncation

2. API and functional correctness

-   [x] check for console logs (warnings, errors)
-   [x] API boundary values
-   [x] different combinations of components - free style
-   [x] change the API values during testing

3. Documentation and Example validations

-   [x] missing API documentation or it is not understandable
-   [x] poor examples
-   [x] Stackblitz works for all examples

4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality

-   [x] the commit message(s) follows the guideline:
        https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
-   [x] tests for the changes that have been done
-   [x] all items on the PR Review Checklist are addressed :
        https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
-   [x] Run npm run build-pack-library and test in external application
-   [x] update `README.md`
-   [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

### CODE REVIEW

File: libs/core/src/lib/slider/slider.component.html [hunk 1]
```diff
@@ -1,16 +1,23 @@
 <div class="fd-slider__inner">
+    <ng-container *ngIf="!_isRange">
```
Comment (platon-rov) [inline libs/core/src/lib/slider/slider.component.html:2]: `*ngIf="!_isRange; else rangeMode"`

File: libs/core/src/lib/slider/slider.component.html [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
+    </ng-container>
+
+    <ng-container *ngIf="_isRange">
+        <ng-container *ngTemplateOutlet="rangeMode"></ng-container>
+    </ng-container>
     <div class="fd-slider__track" (click)="onTrackClick($event)" #track>
         <ng-container *ngIf="!_isRange">
-            <ng-container *ngTemplateOutlet="singleMode"></ng-container>
+            <ng-container *ngTemplateOutlet="singleModeTrack"></ng-container>
         </ng-container>
```
Comment (platon-rov) [inline libs/core/src/lib/slider/slider.component.html:12]: `*ngIf="!_isRange; else rangeModeTrack"`

**Comment by InnaAtanasova** [comment]
The PRs in fund-styles and fund-ngx need scope in their titles. 
feat(core):  .....
feat(platform)....
fix(styles)....

**Review by platon-rov — COMMENTED** [review]
~Value popover doesn't get hide.~
~https://user-images.githubusercontent.com/20265336/169268328-5ec45703-eb13-4d18-b815-5ad95ce12f8c.mov~

EDIT: It's because of the focus state. Then it's alright.

**Review by platon-rov — COMMENTED** [review]
No range slider
image

Range slider
image

Is it intended?

**Comment by InnaAtanasova** [comment]
Code looks good but can't test visually. Could you please fix the deploy preview?

**Comment by N1XUS** [comment]
> Code looks good but can't test visually. Could you please fix the deploy preview?

Hello Inna. You can use this link https://fundamental-ngx-gh--pr8083-pr-chore-slider-hori-dpmuokr2.web.app/

**Comment by N1XUS** [comment]
> No range slider image
> 
> Range slider image
> 
> Is it intended?

Yes. On the first screenshot you have a readonly tooltip. On the second one, editable tooltip

**Comment by mikerodonnell89** [comment]
@N1XUS can you update to the latest `pr/chore/styles-bump-breaking-changes-adoption` and fix conflicts?

**Review by platon-rov — COMMENTED** [review]
Please check failed build phase.

**Comment by g-cheishvili** [comment]
In docs, "Disabled" slider description has typo in it, it should be `[disabled]` not `[disable]`

**Review by g-cheishvili — APPROVED** [review]
I do not see anything wrong

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**